tools/gulp/constants.ts

```diff
@@ -17,6 +29,10 @@ export const HTML_MINIFIER_OPTIONS = {
   removeAttributeQuotes: false
 };
 
+export const UGLIFYJS_OPTIONS = {
+  preserveComments: 'license'
+};
```

Comment:
I think we should just define these where we call uglify if we're only doing it in one place.